### DISCUSSION TITLE
part-ii/2019-86/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#234] [Question comments]
Part II 2019 Logic and Set Theory: Paper 4, Section II, I 

https://questions.tripos.org/part-ii/2019-86/

**Comment by artie2000** [comment]
<details>
<summary>Last bit:</summary>

Set $X_1=\omega_\omega$, and $X_k=\omega_k$ for $k>1$. Then $\bigcup_i X_i=X_1$.

</details>
